[2025-08-18T14:07:56.759+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-18T14:07:04.001858+00:00 [queued]>
[2025-08-18T14:07:56.785+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: ml_pipeline.model_evaluation manual__2025-08-18T14:07:04.001858+00:00 [queued]>
[2025-08-18T14:07:56.789+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-08-18T14:07:56.835+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): model_evaluation> on 2025-08-18 14:07:04.001858+00:00
[2025-08-18T14:07:56.848+0000] {standard_task_runner.py:60} INFO - Started process 3290 to run task
[2025-08-18T14:07:56.857+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'ml_pipeline', 'model_evaluation', 'manual__2025-08-18T14:07:04.001858+00:00', '--job-id', '169', '--raw', '--subdir', 'DAGS_FOLDER/ml_pipeline.py', '--cfg-path', '/tmp/tmprj_wrodr']
[2025-08-18T14:07:56.863+0000] {standard_task_runner.py:88} INFO - Job 169: Subtask model_evaluation
[2025-08-18T14:07:56.983+0000] {task_command.py:423} INFO - Running <TaskInstance: ml_pipeline.model_evaluation manual__2025-08-18T14:07:04.001858+00:00 [running]> on host 06f1b107f378
[2025-08-18T14:07:57.105+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='sarah' AIRFLOW_CTX_DAG_ID='ml_pipeline' AIRFLOW_CTX_TASK_ID='model_evaluation' AIRFLOW_CTX_EXECUTION_DATE='2025-08-18T14:07:04.001858+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-18T14:07:04.001858+00:00'
[2025-08-18T14:08:03.847+0000] {logging_mixin.py:188} INFO - STDOUT:
[2025-08-18T14:08:03.848+0000] {logging_mixin.py:188} INFO -  üèÉ View run traveling-bat-273 at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0/runs/1ec2aba8abc9492f86b30c7fc7647887
üß™ View experiment at: https://dagshub.com/SamMebarek/mlops-som.mlflow/#/experiments/0
Evaluation completed. R2: 0.9998, MAE: 1.1356
[2025-08-18T14:08:03.848+0000] {logging_mixin.py:188} INFO - STDERR:
[2025-08-18T14:08:03.849+0000] {logging_mixin.py:188} INFO -  2025-08-18 14:08:01,044 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/config.yaml
2025-08-18 14:08:01,053 - evaluation.utils.common - INFO - YAML charg√© : /opt/***/project/config/params.yaml
2025-08-18 14:08:01,058 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-18 14:08:01,900 - __main__ - INFO - Loaded processed data: /opt/***/project/data/processed/clean_data.csv (shape=(108060, 14))
2025-08-18 14:08:01,954 - __main__ - INFO - Loaded model: /opt/***/project/models/xgb_model.pkl
2025-08-18 14:08:02,025 - __main__ - INFO - Computed metrics: {'r2_score': 0.9997932180824073, 'mae': 1.135571365098727}
2025-08-18 14:08:02,027 - evaluation.utils.common - INFO - R√©pertoire cr√©√© ou existant : /opt/***/project/evaluation
2025-08-18 14:08:02,030 - evaluation.utils.common - INFO - JSON sauvegard√© : /opt/***/project/evaluation/scores.json
2025-08-18 14:08:02,031 - __main__ - INFO - Metrics saved to: /opt/***/project/evaluation/scores.json
2025/08/18 14:08:02 WARNING mlflow.utils.git_utils: Failed to import Git (the Git executable is probably not on your PATH), so Git SHA is not available. Error: Failed to initialize: Bad git executable.
The git executable must be specified in one of the following ways:
    - be included in your $PATH
    - be set via $GIT_PYTHON_GIT_EXECUTABLE
    - explicitly set via git.refresh(<full-path-to-git-executable>)

All git commands will error until this is rectified.

This initial message can be silenced or aggravated in the future by setting the
$GIT_PYTHON_REFRESH environment variable. Use one of the following values:
    - quiet|q|silence|s|silent|none|n|0: for no message or exception
    - warn|w|warning|log|l|1: for a warning message (logging level CRITICAL, displayed by default)
    - error|e|exception|raise|r|2: for a raised exception

Example:
    export GIT_PYTHON_REFRESH=quiet

2025-08-18 14:08:03,041 - __main__ - INFO - Metrics logged to MLflow.
[2025-08-18T14:08:03.850+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-08-18T14:08:03.862+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=ml_pipeline, task_id=model_evaluation, execution_date=20250818T140704, start_date=20250818T140756, end_date=20250818T140803
[2025-08-18T14:08:03.909+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-08-18T14:08:03.942+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
